<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquationsClass</class>
 <widget class="QMainWindow" name="EquationsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equations</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QLabel" name="aLabel">
      <property name="text">
       <string>a</string>
      </property>
      <property name="buddy">
       <cstring>aEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="bLabel">
      <property name="text">
       <string>b</string>
      </property>
      <property name="buddy">
       <cstring>bEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="combo"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="aEdit"/>
    </item>
    <item row="0" column="0" rowspan="4">
     <widget class="QListWidget" name="list"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="bEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="cEdit"/>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="cLabel">
      <property name="text">
       <string>c</string>
      </property>
      <property name="buddy">
       <cstring>cEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="updateButton">
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="solve">
      <property name="text">
       <string>Solve</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="solution">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="equations.qrc"/>
 </resources>
 <connections/>
</ui>
